src/api/spec/controllers/search_controller_spec.rb

```diff
@@ -5,19 +5,20 @@
   render_views
 
   let!(:user) { create(:confirmed_user, :with_home, login: 'foo') }
```

Comment:
Now you really don't need to use `let!` anymore..

Reply:
done